index.js

```diff
@@ -1,27 +1,67 @@
 var stylus  = require('stylus');
+var path    = require('path');
+var fs      = require('fs');
+
+var stylusUtils = require('stylus/lib/utils');
+
+var stylusLookupIndex = stylusUtils.lookupIndex;
+
+stylusUtils.lookupIndex = function(name, paths, filename){
+  var nodeModuleMatch = paths.find(includePath => includePath.endsWith(name));
+
+  var found = stylusLookupIndex(name, paths, filename);
+  
+  // check for node_modules/<name>
+  if (!found && nodeModuleMatch) {
+    found = stylusLookupIndex(nodeModuleMatch, paths, filename);
+  }
+  if (!found) {
+    console.warn('[derby-stylus] Could not find ' + name + ' in ' + paths.join(', '));
+  }
+  return found;
+};
+
+
+module.exports = function(app, options = {}) {
+  var { includes = [], plugins = [], ...opts } = options;
+  
+  var stylusRequirePath = require.resolve('stylus');
+  var nodeModulesRoot = stylusRequirePath.substring(0, stylusRequirePath.indexOf('/stylus'));
+
+  var includePaths = includes.reduce((acc, moduleId) => {
+    var stylusModulePath = path.join(nodeModulesRoot, moduleId);
+    if (fs.existsSync(stylusModulePath)) {
+      acc.push(stylusModulePath);
+    }
+     return acc;
+  }, []);
+
+  function stylusCompiler(file, filename, options) {
+    var options = { _imports: [], ...options, ...opts };
```

Comment:
The `options` argument here was passed in a specific `loadStylesSync` call:
https://github.com/derbyjs/derby/blob/bcdbeaccb0a0fc0293022ce3574d3f414bcf4f42/src/files.ts#L59-L76

So the more specific call's options should take precedence over the base options provided to the plugin as a whole.